frontend/app/professor/test-simulations/page.tsx

```
@@ -1417,309 +1817,603 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
+                                <div className="text-xs text-gray-500 mb-2">Mention everyone in this scene</div>
+                                {simulationData.current_scene.personas.map((persona) => (
+                                  <div
+                                    key={persona.id}
+                                    className="flex items-center gap-2 p-2 hover:bg-gray-50 rounded cursor-pointer"
+                                    onClick={() => {
+                                      const mentionId = persona.name.toLowerCase().replace(/\s+/g, '_');
+                                      setInput(input.replace(/@[^@]*$/, `@${mentionId} `));
+                                      setShowMentionDropdown(false);
+                                    }}
+                                  >
+                                    <div className="w-6 h-6 bg-gray-200 rounded-full flex items-center justify-center flex-shrink-0">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)